contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -253,10 +253,20 @@ including configuration settings provided by the user,
 or otherwise changed after creation by other ecosystem components (e.g.,
 schedulers, auto-scalers), and is persisted in stable storage with the API
 object. If the specification is deleted, the object will be purged from the
-system. The status summarizes the current state of the object in the system, and
-is usually persisted with the object by automated processes but may be
-generated on the fly. At some cost and perhaps some temporary degradation in
-behavior, the status could be reconstructed by observation if it were lost.
+system.
+
+The status summarizes the current state of the object in the system, and is
+usually persisted with the object by automated processes but may be generated
+on the fly.  As a general guideline, fields in status should be the most recent
+observations of actual state, but they may contain information such as the
+results of allocations or similar operations which are executed in response to
+the object's spec.  See [below](#representing-allocated-values) for more
+details.
+
+Types with spec and status stanzas have distinct authorization scopes for them.
```

Comment:
Maybe "can have"? Since at least for custom resources, the user of the status subresource is optional.

Reply:
I'd suggest "should have" because even for CRDs it's a good idea. :)